index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = App
+
+const jsonwebtoken = require('jsonwebtoken')
+
+function App ({id, privateKey}) {
+  const api = {
+    async appAuth () {
```

Comment:
nit: do we need to make this async? it doesn't appear that we are awaiting on anything

Reply:
yeah good point!